### PR TITLE
Update aws package

### PR DESCRIPTION
**Pull request by JL2210**
Tested on x86_64

### CODE REVIEW

File: packages/aws.rb [hunk 1]
```diff
@@ -3,34 +3,24 @@
...
-    system "mkdir -p #{CREW_DEST_PREFIX}/bin"
-    system "mkdir -p #{CREW_DEST_PREFIX}/share"
-    system "cp #{CREW_PREFIX}/bin/aws #{CREW_DEST_PREFIX}/bin"
-    system "cp -r #{CREW_PREFIX}/share/aws #{CREW_DEST_PREFIX}/share"
+    system "cat requirements-docs.txt >> requirements.txt"
+    system "sed -i 's,-e git://github.com/boto/botocore.git@develop#egg=botocore,botocore,g' requirements.txt"
+    system "sed -i 's,-e git://github.com/boto/s3transfer.git@develop#egg=s3transfer,s3transfer,g' requirements.txt"
+    system "sed -i 's,-e git://github.com/boto/jmespath.git@develop#egg=jmespath,jmespath,g' requirements.txt"
+    system "pip3 install aws -r requirements.txt --upgrade --prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR}"
+    system "python3 setup.py build"
```
Comment (uberhacker) [inline packages/aws.rb]: Can we move all these steps into `def self.build`?

File: packages/aws.rb [hunk 2]
```diff
@@ -3,34 +3,24 @@
...
-    system "chmod +x #{CREW_PREFIX}/bin/aws"
-    system "mkdir -p #{CREW_DEST_PREFIX}/bin"
-    system "mkdir -p #{CREW_DEST_PREFIX}/share"
-    system "cp #{CREW_PREFIX}/bin/aws #{CREW_DEST_PREFIX}/bin"
-    system "cp -r #{CREW_PREFIX}/share/aws #{CREW_DEST_PREFIX}/share"
+    system "cat requirements-docs.txt >> requirements.txt"
+    system "sed -i 's,-e git://github.com/boto/botocore.git@develop#egg=botocore,botocore,g' requirements.txt"
+    system "sed -i 's,-e git://github.com/boto/s3transfer.git@develop#egg=s3transfer,s3transfer,g' requirements.txt"
+    system "sed -i 's,-e git://github.com/boto/jmespath.git@develop#egg=jmespath,jmespath,g' requirements.txt"
+    system "pip3 install aws -r requirements.txt --upgrade --prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR}"
```
Comment (uberhacker) [inline packages/aws.rb]: This doesn't lock down the version.  Try removing this line and use python3 to install instead and see what happens.

File: packages/aws.rb [hunk 3]
```diff
@@ -3,34 +3,24 @@
...
   binary_sha256 ({
-    aarch64: '4b94294abde4f05a2be26b748036400c456322c303ba274c5f3dcc08be879a05',
-     armv7l: '4b94294abde4f05a2be26b748036400c456322c303ba274c5f3dcc08be879a05',
-       i686: '78c0ad96c3d70e0d0124fd458a78a4221477fe659dd3966d3ab14cbd4b9f7fd2',
-     x86_64: 'f8a8b52a993aefe3f601badaba08a676ca53486e83484feda732bc3916636c1b',
   })
 
-  depends_on 'python27' unless File.exists? "#{CREW_PREFIX}/bin/python"
-  depends_on 'unzip' => :build
+  depends_on 'python3' unless File.exists? "#{CREW_PREFIX}/bin/python3"
```
Comment (uberhacker) [inline packages/aws.rb]: Just `depends_on 'python3'` is good enough.

File: packages/aws.rb (hunk 2 above)
Comment (JL2210) [reply packages/aws.rb]: This is python3. These are dependencies. Using the 'develop' versions gives a random crash.

File: packages/aws.rb (hunk 3 above)
Comment (JL2210) [reply packages/aws.rb]: Yeah.

File: packages/aws.rb (hunk 1 above)
Comment (JL2210) [reply packages/aws.rb]: Sure.

Comment (JL2210) [reply packages/aws.rb]: Done.

**Comment by JL2210** [comment]
Is this good?

**Comment by JL2210** [comment]
Is this good?

File: packages/aws.rb [hunk 4]
```diff
@@ -3,34 +3,27 @@
...
   def self.install
-    system "wget https://s3.amazonaws.com/aws-cli/awscli-bundle.zip"
-    system "unzip awscli-bundle.zip"
-    system "awscli-bundle/install -i #{CREW_PREFIX}/share/aws -b #{CREW_PREFIX}/bin/aws"
-    system "chmod +x #{CREW_PREFIX}/bin/aws"
-    system "mkdir -p #{CREW_DEST_PREFIX}/bin"
-    system "mkdir -p #{CREW_DEST_PREFIX}/share"
-    system "cp #{CREW_PREFIX}/bin/aws #{CREW_DEST_PREFIX}/bin"
-    system "cp -r #{CREW_PREFIX}/share/aws #{CREW_DEST_PREFIX}/share"
+    system "pip3 install aws -r requirements.txt --upgrade --prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR}"
```
Comment (uberhacker) [inline packages/aws.rb]: I don't think this line is needed.  If you do this, you certainly do not need the next line.

File: packages/aws.rb [hunk 5]
```diff
@@ -3,34 +3,27 @@
...
+    system "python3 setup.py install --prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR}"
```
Comment (uberhacker) [inline packages/aws.rb]: What happens if you try this?
```
system "python3 setup.py install --prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR}" -r requirements.txt
```

Comment (JL2210) [reply packages/aws.rb]:
```
Unknown option: -r
usage: python3 [option] ... [-c cmd | -m mod | file | -] [arg] ...
Try `python -h' for more information.
```

Comment (JL2210) [reply packages/aws.rb]: The `python3 setup.py install` command installs the package. `pip3 install aws -r requirements.txt ...` tracks it within pip.

File: packages/aws.rb (hunk 4 above)
Comment (JL2210) [reply packages/aws.rb]: pip3 installs an old version of aws. This is versioned from the git repo.

**Pull request merged**